R/stat-boxplot.r

```diff
@@ -4,13 +4,13 @@
 #' @section Computed variables:
```

Comment:
I think it would be helpful to add a sentence stating that the computed variables depend on the orientation.

Reply:
Sure.